jupyter-book/reproducibility/reproducibility.ipynb

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
<p>Switch the order of the first and second section here. First the issue and motivation, then the definition of things to make them clearer and then how we plan to tackle it (aka explain things here). Especially "Therefore, our goal is to provide the necessary tools that should propel you on the spectrum towards full reproducibility. " is too early in this section.</p><p> </p><p>"Everyone strives to make their projects reproducible. " HA, you wish. No. I'd rather state that everyone SHOULD make their projects reproducible. But you did that already. So remove this sentence.</p><p> </p><blockquote>Consequently, there are many reproducible tools and advice out there{cite}<code>Alston_Rick_2021</code></blockquote><p>Can we add some more citations here? I am sure that there is a lot more out there.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>